packages/toolkit/src/codewhisperer/models/model.ts

```diff
@@ -252,6 +252,7 @@ export enum BuildSystem {
 export class ZipManifest {
     sourcesRoot: string = 'sources/'
     dependenciesRoot: string | undefined = 'dependencies/'
+    buildLogs: string = 'build-logs.txt'
```

Comment:
Small change: add this entry to our `manifest.json` since backend is expecting it to be there.